puppet/modules/slave/templates/test_pull_requests_foreman_ansible.json.erb

```diff
@@ -0,0 +1,49 @@
+{
+    "proxy_host": null,
+    "proxy_port": null,
+    "jenkins_host": "http://ci.theforeman.org/",
+    "jenkins_url": "http://ci.theforeman.org/",
+    "hub_config": "~/.config/hub",
+    "github_user": "theforeman",
+    "bot_github_user": "<%= github_user -%>",
+    "irc_channel": "#theforeman-dev",
+    "not_mergable_cache": "~/pr_tests/cache/test_pull_request_foreman_ansible_not_mergable",
+    "repo_groups": {
+        "foreman_ansible": "foreman_ansible"
+    },
+    "settings": {
+        "test_settings": {
+            "name": "test",
+            "test_prefix": "Test Results:",
+            "allow_multiple": true,
+            "run_without_trigger": true,
+            "branches": {
+                "*": {
+                    "jenkins_job_name": "test_plugin_pull_request",
+                    "downstream_job_name": "test_plugin_foreman_ansible_master",
```

Comment:
This job testing master doesn't currently exist, you'll need to create it first (step 2 at http://projects.theforeman.org/projects/foreman/wiki/Jenkins#Adding-a-new-Foreman-plugin) or change to `true` to not test master at all.